easybuild/easyconfigs/b/bpp-seq/bpp-seq-2.4.1-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,27 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'CMakeMake'
+
+name = "bpp-seq"
+version = '2.4.1'
+
+homepage = 'https://github.com/BioPP/bpp-seq'
+description = """Bio++ Sequence Library"""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+github_account = 'BioPP'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['dbfcb04803e4b7f08f9f159da8a947c91906c3ca8b20683ac193f6dc524d4655']
+
+builddependencies = [('CMake', '3.13.3')]
+
+dependencies = [('bpp-core', '2.4.1')]
+configopts = ' -Dbpp-core_DIR=${EBROOTBPPMINCORE}/lib64/cmake/bpp-core/ '
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ["lib64"]
```

Comment:
@omula I would make this a bit more specific:

```python
sanity_check_paths = {
    'files': ['lib64/libbpp-seq.a', 'lib64/libbpp-seq.%s' % SHLIB_EXT],
    'dirs': ['include/Bpp']
}

Reply:
done